org/pr/milestone.ts

```diff
@@ -22,7 +22,8 @@ export default async () => {
 
         // Skip for PRs for wip features unless the PR is against "develop" or "release/x.x" branches
         const wipFeature = githubLabels.some(label => label.name.includes("Part of a WIP Feature"));
-        if (!targetsDevelop && !targetsRelease && wipFeature) {
+        const donotmergeFeature = githubLabels.some(label => label.name.includes("status: do not merge"));
```

Comment:
```suggestion
        const donotmergeFeature = githubLabels.some(label => label.name.includes("status: feature-flagged"));
```
Or whatever you end up deciding renaming that label after your previous comment.

> I have another question for you, would you prefer a different name for that label? Olivier also expressed concern about the naming for this label and you're the 2nd person to raise the issue

Regarding that btw, I'm fine to keep `status: feature-flagged` or to pick a name that would convey the meaning that it's "a feature that is developed in incremental steps across multiple sprints" and "is currently not visible to end user in release builds so if there's a milestone set it's just an estimate and it's ok to push it to next sprint if not ready in time for this one". Since that's the whole meaning I want us to give to that label, and also the whole point of this Peril check here (if it's such a feature with the characteristics I wrote in quotes above, then it's the only case where it's ok to have a milestone on the PR).